src/planwise/client/current_project/handlers.cljs

```diff
@@ -48,20 +48,45 @@
     :transport :facilities-with-demand))
 
 (defn- project-loaded [db project-data]
-  (dispatch [:current-project/fetch-facility-types])
-  (dispatch [:regions/load-regions-with-geo [(:region-id project-data)]])
-  (db/new-viewmodel project-data))
+  (let [new-db (db/new-viewmodel project-data)]
+    (dispatch [:current-project/fetch-facility-types])
+    (dispatch [:regions/load-regions-with-geo [(:region-id project-data)]])
+    (if (:is-new? project-data)
+      (assoc-in new-db [:wizard :set] true)
+      new-db)))
```

Comment:
Because that's the default value it has on the db, if it's not changes it stays the same.